includes/manage_pages.inc

```diff
@@ -739,6 +739,10 @@ function islandora_paged_content_zipped_upload_form_submit(array $form, array &$
       'model' => $form_state['values']['model'],
       'file_count' => $file_count,
       'book_label' => $book_label,
+      'ignored_derivs' => array(
+        'ocr' => $form_state['values']['ignore_ocr'],
+        'hocr' => $form_state['values']['ignore_hocr'],
+      ),
```

Comment:
If they don't have `islandora_ocr` installed, this will explode I think. Probably needs an `isset()` or the same test from before 
```
if (module_exists('islandora_ocr') && in_array('ocr', $derivatives)) {
```